commcare-cloud/README.md

```diff
@@ -0,0 +1,84 @@
+# Set up commcarehq-ansible
+
+Clone the commcarehq-ansible repo (suggested location alongside commcare-hq):
+
+```
+git clone https://github.com/dimagi/commcarehq-ansible.git
+cd commcarehq-ansible
+```
+
+Now make a virtualenv for ansible:
+
+```
+mkvirtualenv ansible
+```
+
+If you want `workon ansible` to always bring you to this directory, then you can also run
+
+```
+pwd > ~/.virtualenvs/ansible/.project
+```
+
+at this time.
+
+**Note**: If you already have commcarehq-ansible cloned, then just enter that directory
+and update it with
+
+```
+git pull
+```
+
+# Install commcare-cloud
+
+```
+workon ansible
+```
+
+```
+pip install pip --upgrade
+```
+
+```
+pip install -e commcare-cloud
+pip install -r ansible/requirements.txt  # to be able to run ansible commands
+ansible-galaxy install -r ansible/requirements.yml  # to be able to run ansible commands
```

Comment:
does this make sense locally too?

Reply:
Yeah it can; I sometimes run simple commands from my computer. It's not strictly necessary for everything, but it's nice to get everyone doing all the installs so there's some kind of consistency.